### PR TITLE
scope dependenies (WP-648)

### PR DESCRIPTION
**Pull request by vsolovei-smartling**
Build process changed to produce dependencies with changed namespace

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ### COMPOSER ###
 composer
 
+inc/lib
```
Comment (PavelLoparev) [inline .gitignore:4]: What is this folder? I don't have in master and in this branch as well

File: inc/Smartling/ContentTypes/AutoDiscover/PostTypes.php [hunk 2]
```diff
@@ -2,9 +2,6 @@
 
 namespace Smartling\ContentTypes\AutoDiscover;
 
-use Psr\Log\LoggerInterface;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
```
Comment (PavelLoparev) [inline inc/Smartling/ContentTypes/AutoDiscover/PostTypes.php:6]: In previous files you changed this to `Smartling\Vendor\...` namespace dependencies. Why just remove here?

File: inc/Smartling/WP/View/ContentEditJob.php [hunk 3]
```diff
@@ -399,6 +401,8 @@ function (i, e) {
                     }
                 }
 
+                console.log(result);
+
```
Comment (PavelLoparev) [inline inc/Smartling/WP/View/ContentEditJob.php:405]: Seems like debug leftover. Let's remove?

File: inc/autoload.php [hunk 4]
```diff
@@ -13,10 +13,9 @@
  * Generates an \Exception if composer autoload file not found.
  */
 $composerAutoLoadScriptFile
-	= SMARTLING_PLUGIN_DIR . DIRECTORY_SEPARATOR . 'inc' . DIRECTORY_SEPARATOR . 'third-party' . DIRECTORY_SEPARATOR . 'autoload.php';
+	= SMARTLING_PLUGIN_DIR . DIRECTORY_SEPARATOR . 'inc' . DIRECTORY_SEPARATOR . 'lib' . DIRECTORY_SEPARATOR . 'autoload.php';
```
Comment (PavelLoparev) [inline inc/autoload.php:16]: Oh, so we changed third-party with lib folder. Ok

File: readme.txt [hunk 5]
```diff
@@ -62,6 +62,10 @@ Additional information on the Smartling Connector for WordPress can be found [he
 3. Track translation status within WordPress from the Submissions Board. View overall progress of submitted translation requests as well as resend updated content.
 
 == Changelog ==
+= 2.8.0 =
+* Scoped dependencies to prevent conflicts with other installed versions of GuzzleHttp and Symfony libraries
+* Fixed content not being added to upload from post widget
```
Comment (PavelLoparev) [inline readme.txt:67]: Oh, again. 2 issues in one PR (which contains only one Jira ticket id which is confusing)

File: tests/MonologWrapper/resources/LevelLoggerTests/services_alert.yml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 services:
   handler:
-    class: Monolog\Handler\NullHandler
+    class: Smartling\Vendor\Monolog\Handler\NullHandler
```
Comment (PavelLoparev) [inline tests/MonologWrapper/resources/LevelLoggerTests/services_alert.yml:3]: Is this tool (namespacer) fixes namespaces in yml configs as well or you change these manually?

**Review by PavelLoparev — CHANGES_REQUESTED** [review]

File: inc/Smartling/WP/View/ContentEditJob.php (hunk 3 above)
Comment (vsolovei-smartling) [reply inc/Smartling/WP/View/ContentEditJob.php]: Removed irrelevant

**Review by PavelLoparev — APPROVED** [review]

**Pull request merged**